### ISSUE TITLE
notebooks: Notebook posts no longer have support for markdown

### ISSUE THREAD

**Issue by thelifeandtimes** [tloncorp/landscape-apps#1610]
**Describe the bug, optionally with screenshots**
I cannot use markdown to format the contents of a notebook post. This means that I cannot do tables, different level headers, or any other functionality that was available in the previous version of notebooks. 

**To Reproduce**
1. Go to a new notebook post
2. Try to create a post with multiple levels of content and tables
3. See error

**Expected behavior**
I understand the desire for supporting basic front end formatting for things like bold, underlines, codeblocks, etc. But this change is a regression in functionality. My expectation would be that at the very least there would be an option to just write pure (or some modified version of) markdown and have it render correctly. Instead I have old posts and other markdown content that would require total restructuring in order to move it into a groups notebook and have it be human readable.

**Additional context:**
- Ship type: Planet
- Instance type: Self-Hosted on ubuntu 22.04
- OS/browser: Brave

**Comment by yapishu** [comment]
I am also having issues with my notebooks that were migrated -- some of them had chunks of content stop displaying, and when I try to modify them, I get an error message. In another of my notebooks, if I try to edit the content, the WYSIWYG editor returns an empty post.

**Comment by arthyn** [comment]
Sorry about that, we have some issues with old content not being handled and that clears the editor unfortunately.

@thelifeandtimes you should be able to use headings still, but we don't have support for tables yet for sure.

**Comment by thelifeandtimes** [comment]
@arthyn what is the expected way for using headings because `#` doesn't seem to do it?

**Comment by arthyn** [comment]
You have to put a space after the amount of `#`'s you want

**Comment by thelifeandtimes** [comment]
This does not work:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/82244264/210457881-1faa6db9-33ec-406d-9c1f-a40f0a293431.png">

**Comment by yapishu** [comment]
@thelifeandtimes it's temperamental but try clearing the line and retyping it, it won't register if there's text after the `#`s

**Comment by thelifeandtimes** [comment]
So, in effect, that makes it impossible to copy and paste markdown in that was written elsewhere? I think it is a common user story at present that people draft these things elsewhere and paste them into notebooks. Would be nice to continue supporting that. Am I correct that this change has to do with some desires around an urbit-specific AST?

**Comment by arthyn** [comment]
Yeah right now you can't copy+paste markdown. We chose to not support markdown for a variety of reasons. One of which is that we wanted to avoid having to write markdown parsers or use existing ones on the frontend, and because it's much better to have a hoon side AST so other clients and consumers of content externally could read the AST instead of having to have a markdown parser.

**Comment by thelifeandtimes** [comment]
in case it helps, @tacryt-socryp was working on something in this vein: https://urbit.org/grants/hoon-markdown-library not sure where it lives or if there is anything useful but figured I'd raise it.

**Comment by vcavallo** [comment]
> Yeah right now you can't copy+paste markdown

Are there plans to support copying/pasting markdown into posts (as well as copying post content _as markdown__) in the future? It would make manual inter-op between other platforms much more possible.

**Comment by jamesacklin** [comment]
We're looking at several approaches this weekend.

**Comment by jamesacklin** [comment]
Tracking internally as LAND-368
